### PR TITLE
3.3.0 website release

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: src/homepage/_posts/2019-02-12-james-3.3.0.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- RecipientRewriteTable Aliases and associated WebAdmin routes
+
+### Fixed
+
+- MAILBOX-350 Potential invalid UID <-> MSN mapping upon IMAP COPY
+- Possibility to better zoom in Grafana boards
+- default ElasticSearch shards & replica configured values
+- Move & copy batch sizes are now loaded from configuration
+
+### Changed
```
Comment (trantienduchn) [inline src/homepage/_posts/2019-02-12-james-3.3.0.markdown]: how about adding one more item of Reactor migration?

**Comment by chibenwa** [comment]
Call for vote:

```
Subject: Call for vote: Apache James 3.3.0

Hi,

I would like to propose the 3.3.0 release of the Apache James server.

Furthermore, you can see changes proposed to the website at the occasion
of that release, as well as communication on this GitHub pull request:
https://github.com/linagora/james-project/pull/2169

You can find:

 - The maven release staged in http://repository.apache.org/ as the artifact
#1042
 - 3.3.0 sources and binaries are uploaded on Apache dist. 
 - The changelog for 3.3.0: https://github.com/chibenwa/james-project/blob/3.3.0-website-release/CHANGELOG.md
 - The compatibility instructions/upgrade recommendation: https://github.com/chibenwa/james-project/blob/3.3.0-website-release/upgrade-instructions.md

Voting rules:
 - This is a majority approval: this release may not be vetoed.
 - A quorum of 3 binding votes is required
 - The vote starts at XXXXX XXth of November 2018, 2am UTC
 - The vote ends at XXXXX XXth of November 2018, 2am UTC

You can answer to it just with +1 and -1. Down-votes may be motivated.

Cheers,

Benoit Tellier
```

File: src/homepage/_posts/2019-02-12-james-3.3.0.markdown (hunk 1 above)
Comment (chibenwa) [reply src/homepage/_posts/2019-02-12-james-3.3.0.markdown]: Sounds a good idea

**Comment by chibenwa** [comment]
Tweet:

```
Apache James 3.3.0 is out! 

Check it out here: {link_to_the_post}

Many thanks to all contributors!
```

Announce:

```
Subject: [ANNOUNCE] Apache James 3.3.0 released
To: announce@apache.org, server-user, server-dev

The Apache James community is proud to announce you the 3.3.0 release of the Apache James server. 

Apache James provides a complete, stable, secure and extendable Mail Servers running on the JVM. In this release, we worked hard toward providing a highly scalable mail server. More details about Apache James can be found at: https://james.apache.org/

The release is available for download at: https://james.apache.org/download.cgi#Apache_James_Server

Changes from 3.3.0 includes:
Tweet:

```
Apache James 3.2.0 is out! 

Check it out here: {link_to_the_post}

Many thanks to all contributors!
```

Announce:

```
Subject: [ANNOUNCE] Apache James 3.2.0 released
To: announce@apache.org, server-user, server-dev

The Apache James community is proud to announce you the 3.2.0 release of the Apache James server. 

Apache James provides a complete, stable, secure and extendable Mail Servers running on the JVM. More details about Apache James can be found at: https://james.apache.org/

The release is available for download at: https://james.apache.org/download.cgi#Apache_James_Server

Changes from 3.2.0 includes:

- Metrics for BlobStore
- New Guice product using Cassandra RabbitMQ ElasticSearch, OpenStack Swift and optional LDAP dependency (experimental)
- JPA SMTP dockerFile 
- Listing healthchecks
- Configuring the ElasticSearch clusterName
- Logging and Metrics now supports Elasticsearch 6 (previously only Elasticsearch 2 was supported)
- Implementation of the RabbitMQ EventBus
- DeadLetter APIs and memory implementation for storing events that failed delivery
- RecipientRewriteTable Aliases and associated WebAdmin routes

- Possibility to better zoom in Grafana boards
- default ElasticSearch shards & replica configured values
- Move & copy batch sizes are now loaded from configuration
- Migration of concurrency management to reactor for several backends

- WebAdmin ReIndexing API had been reworked
- Docker images are now using a JRE instead of a JDK

- Drop HBase and JCR components (mailbox and server/data).

We would like to thank all of contributors who made the release possible and
to invite others interested in helping out to engage the community on the dev
and users lists!
```

We would like to thank all of the contributors who made the release possible and
to invite others interested in helping out to engage the community on the dev
and users lists!
```

File: src/site/xdoc/server/config-mailrepositorystore.xml [hunk 2]
```diff
@@ -89,7 +89,7 @@
     
     <subsection name="MBox Repositories">
 
-     <p>Warning: deprecated. This feature will be removed as part of James release 3.2.0 unless it is maintained.
+     <p>Warning: deprecated. This feature had been removed as part of James release 3.2.0.
```
Comment (rouazana) [inline src/site/xdoc/server/config-mailrepositorystore.xml]: then let's remove this section

File: src/site/xdoc/server/install.xml [hunk 3]
```diff
@@ -29,11 +29,11 @@
 
   <subsection name="Java Virtual Machine">
   
-    <p>James 3.1 requires a Java Runtime Environment. Java version 1.8 is required to run the
+    <p>James 3.3.0 requires a Java Runtime Environment. Java version 1.8 is required to run the
        James application. The exact JREs available depends on the platform. In addition, the environment variable JAVA_HOME should be set to
        the JRE home directory before running James.</p>
     
-    <p>James 3.1 has been successfully tested on OpenJDK</p>
+    <p>James 3.3.0 has been successfully tested on OpenJDK</p>
```
Comment (blackheaven) [inline src/site/xdoc/server/install.xml]: Maybe the version could be specified

**Comment by aduprat** [comment]
Was :green_apple: before CI upgrade

**Comment by aduprat** [comment]
Was :green_apple: before CI restart.

**Comment by aduprat** [comment]
test this please

**Pull request merged**